src/app/permissions/add-edit-project-survey/add-edit-project-survey.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="spinner-container" *ngIf="loading">
+  <div class="spinner-new rotating"></div>
+</div>
+
+<nav *ngIf="!loading" class="action-container" aria-label="breadcrumb">
```

Comment:
@Baelx Just wondering if the breadcrumb could be its own component, see it in multiple spots,  i know it is slightly different... Just a thought.